app/views/uploads/_population_criteria.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="panel panel-default <%=population[:type]%>_children">
+  <div class="panel-heading">
+    <h4 class="panel-title panel-population">
+      <a data-toggle="collapse" href="#collapse_<%=measure[:cms_id]%>_<%=measure[:sub_id]%>_<%=population[:hqmf_id]%>">
+        <div>
+          <% title = population[:title] %>
+          <% if (population[:type] == 'OBSERV') then title = 'Observations' end %>
+          <span class="population-label <%=title%> rationale eval-<%=@rationale[population[:type]]%>"><b><%=title%></b><span class="sr-only sr-highlight-status"></span></span>
+          <div class="pull-right">
+            <i class="fa fa-lg fa-angle-down toggle-icon" aria-hidden="true"></i>
+            <span class="sr-only">toggle <%=title%></span>
+          </div>
+        </div>
+      </a>
+    </h4>
+  </div>
+
+  <div id="collapse_<%=measure[:cms_id]%>_<%=measure[:sub_id]%>_<%=population[:hqmf_id]%>" class="panel-collapse collapse in">
+    <div class="panel-body">
+      <% if population[:preconditions] %>
+        <% rootPrecondition = population[:preconditions][0] %>
+        <% if rootPrecondition[:preconditions] %>
+          <% rootPrecondition[:preconditions].each do |precondition| %>
+            <%= render partial: 'precondition', locals: {precondition: precondition, parentPrecondition: rootPrecondition, parentNegation: rootPrecondition[:negation], measure: measure} %>
+          <% end %>
+        <% else %>
+            <li>
+              <% if population[:aggregator] %>
+                <%= population[:aggregator] %> of
+              <% end %>
+              <%= render partial: 'data_criteria', locals: {reference: rootPrecondition[:reference], expandVariable: false, measure: measure, tag: "span"} %>
+            </li>
+          </ul>
```

Comment:
I don't see a matching opening `<ul>` tag. Is there supposed to be one somewhere here?